src/votekit/election_types.py

```diff
@@ -1,117 +1,140 @@
 from .profile import PreferenceProfile
 from .ballot import Ballot
-from .models import Outcome
+from .election_state import ElectionState
 from typing import Callable
 import random
 from fractions import Fraction
 from copy import deepcopy
 
-
+##We're not passing winner_votes to outcome right now
+# If we want to track which candidate each ballot is ultimately going to, we need to do more.
+# TODO:
+# 1. winner_votes to election_state is supposed to have all the ballots in the
+# initial profile that ended up going to any elected candidate.
+# Bit tricky since all of our methods work with current state of profile not the initial
+# 2. integrate Cincinatti transfer
 class STV:
     def __init__(self, profile: PreferenceProfile, transfer: Callable, seats: int):
-        self.profile = profile
-        self.transfer = transfer
-        self.elected: set = set()
-        self.eliminated: set = set()
-        self.seats = seats
-        self.threshold = self.get_threshold()
+        self.__profile = profile
+        self.transfer: Callable = transfer
+        self.seats: int = seats
+        self.election_state: ElectionState = ElectionState(
```

Comment:
note: you don't have to annotate these since the type can be inferred by the RHS of the equals